documentation/docs/steps/cloudFoundryCreateService.md

```diff
@@ -2,6 +2,17 @@
 
 ## ${docGenDescription}
 
+## Prerequisites
+
+* This step is for creating one or multiple Services in Cloud Foundry.
+* Cloud Foundry API endpoint, Organization, Space and user are available
+* If you chose to create a single Service the Service Instance Name, Plan Broker of the Service to be created have to be available
+* Credentials have been configured in Jenkins with a dedicated Id
+* You can set the optional `cfCreateServiceConfig` flag to configure the Service creation with your respective JSON configuration. The JSON configuration can either be an in-line JSON string or the path a dedicated JSON configuration file containing the JSON configuration. If you chose a dedicated config file, you must store the file in the same folder as your `Jenkinsfile` that starts the Pipeline in order for the Pipeline to be able to find the file. Most favourable SCM is Git.
```

Comment:
In my opinion, a lot of these points don't belong in the section "Prerequisites". I think it would be beneficial to restructure them

Reply:
Restructured it mainly in the long description in the `yaml` file. What do you think?